### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mercihabam**

### CODE REVIEW

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
### Good job on the project
Please check out [this](https://musayann.notion.site/BEM-reviews-highlights-0d16ab192f134555a6f9ecf64a230498), revise your work accordingly, and re-request reviews.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    </section>
+
+    <section class="qualities">
+        <div class="qualities__title">High-quality</div>
+        <div class="qualities__main-title">We have the Best Solution for your Business</div>
+        <div class="qualities__items">
+            <div class="qualities__item">
+                <img src="./assets/icons/sec.svg" alt="" class="qualities__icon">
+                <div class="qualities__name">High security to protect from piracy</div>
+                <div class="qualities_desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
```
Comment (Nkaka23dev) [inline index.html]: This is not BEM

File: index.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            </div>
+
+            <div class="qualities__item">
+                <img src="./assets/icons/like.svg" alt="" class="qualities__icon">
+                <div class="qualities__name">Premium quality performance</div>
+                <div class="qualities_desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
```
Comment (Nkaka23dev) [inline index.html]: Same here not BEM

File: index.html [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+            </div>
+
+            <div class="qualities__item">
+                <img src="./assets/icons/headphones.svg" alt="" class="qualities__icon">
+                <div class="qualities__name">Full time customer support - 24/7</div>
+                <div class="qualities_desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
```
Comment (Nkaka23dev) [inline index.html:59]: Not BEM

File: index.html [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+    </section>
+
+    <section class="why-us">
```
Comment (Nkaka23dev) [inline index.html]: Why-us is not a agood name for a block in BEM, you can name it like work or business

File: index.html [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+                </div>
+                <div class="why-us__detail">
+                    <img src="./assets/icons/arrow.svg" alt="">
+                    <div class="why-us__list-item">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="stats">
```
Comment (Nkaka23dev) [inline index.html]: stats name is not descriptive, give it a meaningfull name

File: index.html [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+        <div class="stats__details">
+            <div class="stats__title">More impressions, more conversions</div>
+            <div class="stats__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.</div>
+            <button class="stats__trial-btn">Get Free trial</button>
```
Comment (Nkaka23dev) [inline index.html]: stats__trial-btn same here

File: index.html [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+            <div class="discover-more__details">
+                <div class="discover-more__main-title">Analyze your sales and marketing leads</div>
+                <div class="discover-more__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.</div>
+            </div>
+            <div class="discover-more__items">
+                <div class="discover-more__item">
+                    <img src="./assets/icons/sales.svg" alt="">
+                    <div class="discover-more__item-content">
+                        <div class="discover-more__item-name">Sales Tracking</div>
+                        <div class="discover-more__item-desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</div>
```
Comment (Nkaka23dev) [inline index.html]: discover-more__item-desc, is not descriptive desc is meaningful I suggest you change it to be discover-more__description

File: index.html [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+                    </div>
+                </div>
+                <div class="discover-more__item">
+                    <img src="./assets/icons/manager.svg" alt="">
+                    <div class="discover-more__item-content">
+                        <div class="discover-more__item-name">Project Management</div>
```
Comment (Nkaka23dev) [inline index.html:121]: same here.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+                        <div class="discover-more__item-desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</div>
```
Comment (Nkaka23dev) [inline index.html]: same here desc should be changed

File: index.html [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+                    </div>
+                </div>
+                <div class="discover-more__item">
+                    <img src="./assets/icons/reporter.svg" alt="">
+                    <div class="discover-more__item-content">
+                        <div class="discover-more__item-name">Activity Report</div>
+                        <div class="discover-more__item-desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</div>
```
Comment (Nkaka23dev) [inline index.html]: Here also

File: index.html [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+    </section>
+
+    <section class="features">
+        <h2 class="features__title">Features</h2>
+        <div class="features__container">
+            <div class="features__item">
+                <img src="./assets/icons/feat1.svg" alt="" class="features__icon">
+                <div class="features__name">Advanced 256-bit encryption</div>
+                <div class="features__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
```
Comment (Nkaka23dev) [inline index.html]: features__desc should be feature__description.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,240 @@
...
+            </div>
+
+            <div class="features__item features__item--box">
+                <img src="./assets/icons/feat2.svg" alt="" class="features__icon">
+                <div class="features__name">Simple collaboration tools</div>
+                <div class="features__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
```
Comment (Nkaka23dev) [inline index.html:149]: here also

File: index.html [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+
+            <div class="features__item features__item--box">
+                <img src="./assets/icons/feat3.svg" alt="" class="features__icon">
+                <div class="features__name">Customizable AI features</div>
+                <div class="features__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
```
Comment (Nkaka23dev) [inline index.html]: and here

File: index.html [hunk 14]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+    </section>
+
+    <section class="clients">
+        <h2 class="clients__title">The stunning results our
+            customers have experienced</h2>
+
+        <div class="clients__cards">
+            <div class="clients__card">
+                <div class="clients__msg">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</div>
```
Comment (Nkaka23dev) [inline index.html]: clients__msg should be clients__message

File: index.html [hunk 15]
```diff
@@ -0,0 +1,240 @@
...
+                <img src="./assets/img/c1.png" alt="" class="clients__avatar">
+                <div class="clients__name">Ron Wood</div>
+                <div class="clients__job">CEO</div>
+            </div>
+            <div class="clients__card">
+                <div class="clients__msg">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</div>
```
Comment (Nkaka23dev) [inline index.html]: Same here

File: index.html [hunk 16]
```diff
@@ -0,0 +1,240 @@
...
+                <img src="./assets/img/c2.png" alt="" class="clients__avatar">
+                <div class="clients__name">Mark Mason</div>
+                <div class="clients__job">Marketing Manager</div>
+            </div>
+            <div class="clients__card">
+                <div class="clients__msg">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</div>
```
Comment (Nkaka23dev) [inline index.html]: And here

File: index.html [hunk 17]
```diff
@@ -0,0 +1,240 @@
...
+                <img src="./assets/img/c3.png" alt="" class="clients__avatar">
+                <div class="clients__name">Sam Preston</div>
+                <div class="clients__job">CTO</div>
+            </div>
+        </div>
+    </section>
+
+    <div class="call-to-act">
```
Comment (Nkaka23dev) [inline index.html]: call to act is not a good name it should atleast be named pricing

File: pricing.html [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+                        <img src="./assets/icons/pricing/checked.svg" alt="" class="pricing__checked-icon">
+                        <div class="pricing__feature-name">3GB storage</div>
+                    </div>
+                    <div class="pricing__feature">
+                        <img src="./assets/icons/pricing/checked.svg" alt="" class="pricing__checked-icon">
+                        <div class="pricing__feature-name">Ads Preview</div>
+                    </div>
+                </div>
+
+                <div class="pricing__description mt-8">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</div>
```
Comment (Nkaka23dev) [inline pricing.html]: mt-8 should be in css file instead of html

File: product.html [hunk 19]
```diff
@@ -0,0 +1,221 @@
...
+            <div class="prod-hero__btns">
+                <button class="prod-hero__start-btn">Get Started</button>
+                <button class="prod-hero__contact-btn">Contact Us</button>
+            </div>
+        </div>
+        <div class="prod-hero__img-container">
+            <img src="./assets/img/dashb.png" alt="" class="prod-hero__img">
+        </div>
+        <img src="./assets/img/product/hero-gradient.svg" alt="" class="prod-hero__gradient">
+    </section>
```
Comment (alain-kubwayo) [inline product.html]: You should use a meaningful name here. `prod`

File: product.html [hunk 20]
```diff
@@ -0,0 +1,221 @@
...
+
+    <section class="p-20 flex justify-between">
```
Comment (alain-kubwayo) [inline product.html]: Consider adding these classes to the CSS file with`@apply` directive and call a class instead to keep your HTML clean.

File: product.html [hunk 21]
```diff
@@ -0,0 +1,221 @@
...
+        <img src="./assets/img/partner1.svg" alt="">
+        <img src="./assets/img/partner2.svg" alt="">
+        <img src="./assets/img/partner3.svg" alt="">
+        <img src="./assets/img/partner4.svg" alt="">
+        <img src="./assets/img/partner5.svg" alt="">
+    </section>
+
+    <section class="prod-features">
```
Comment (alain-kubwayo) [inline product.html]: Same naming issue here too.  Adjust `prod-..` name to be more descriptive for this section.

File: product.html [hunk 22]
```diff
@@ -0,0 +1,221 @@
...
+                <div class="insights__gradient-title">Time tracker</div>
+                <div class="insights__title">Track your project performance</div>
+                <div class="insights__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.</div>
+            </div>
+        </div>
+    </section>
+    
+    <section class="insights">
+        <img src="./assets/img/product/lead-ins-grad.svg" alt="" class="absolute right-0 -top-80 z-0">
+        <div class="insights__content relative z-10">
```
Comment (alain-kubwayo) [inline product.html]: You should add these classes to the CSS file and use a custom class name instead that follow BEM convention to be consistent.

File: product.html [hunk 23]
```diff
@@ -0,0 +1,221 @@
...
+                <div class="insights__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.</div>
+            </div>
+            <div class="insights__details">
+                <img src="./assets/img/product/lead-ins.png" alt="" class="insights__img">
+            </div>
+        </div>
+    </section>
+
+    <section class="insights">
+        <div class="grid grid-cols-[45%,1fr] items-center gap-20 p-20 relative z-10">
```
Comment (alain-kubwayo) [inline product.html]: Please move these classes to the CSS file.

File: product.html [hunk 24]
```diff
@@ -0,0 +1,221 @@
...
+    </section>
+
+    <div class="view-pricing">
+        <div class="view-pricing__content">
+            <h2 class="view-pricing__title">Are you ready to grow your business with us?</h2>
+            <div class="view-pricing__subtitle">Lorem ipsum dolor sit amet, consetetur
+                Subscribe to our Nesadipscing elitr, sed diam nonumy.</div>
+            <button class="view-pricing__btn">View Pricing</button>
+        </div>
+        <img src="./assets/img/footer-blur-bg.png" alt="" class="view-pricing__blur-bg">
```
Comment (alain-kubwayo) [inline product.html]: `view-pricing__blur-bg` is kind of confusing. Try maybe `view-pricing__blurred-background` or similar descriptive names.

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
### Good job
---
Few minor adjustments to make.

File: product.html [hunk 25]
```diff
@@ -127,8 +127,8 @@
         </div>
     </section>
 
-    <section class="insights">
-        <div class="grid grid-cols-[45%,1fr] items-center gap-20 p-20 relative z-10">
+    <section class="insights">grid grid-cols-[45%,1fr] items-center gap-20 p-20 relative z-10
```
Comment (alain-kubwayo) [inline product.html]: These classes shouldn't be here. Consider moving them to the external CSS file or simply delete them if you're not using them and already have `insights` block.

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
### Good job working on the requested changes. 
---
Address the following minor issue.

File: style.css [hunk 26]
```diff
@@ -0,0 +1,377 @@
...
+    .product-hero__contact-btn{
+        @apply bg-transparent text-primary border-2 border-blue-100 rounded-xl active:bg-primary active:text-white active:border-transparent py-4 px-10 outline-none font-bold transition-all
+    }
+    .c-title, .work__main-title, .statistics__title, .discover-more__main-title, .features__title, .features__name, .clients__title, .view-pricing__title, .product-features__title, .insights__title{
+        @apply font-bold text-[48px] leading-[120%]
+    }
+    .c-subtitle, .work__description, .statistics__subtitle, .discover-more__subtitle, .features__description, .view-pricing__subtitle, .insights__subtitle, .pricing-hero__subtitle, .product__features-description{
+        @apply text-base opacity-90
+    }
+    .s-hero__gradient, .hero__gradient, .product-hero__gradient{
```
Comment (Nkaka23dev) [inline style.css:125]: use tailwind class instead of custom csss

File: style.css [hunk 27]
```diff
@@ -0,0 +1,377 @@
...
+        position: absolute;
+        top: -80px;
+        z-index: -1;
+    }
+    .c-input, .footer__input{
+        @apply bg-white border-none outline-none
+    }
+    .c-gradient-title, .qualities__title, .work__title, .discover-more__title, .insights__gradient-title{
```
Comment (Nkaka23dev) [inline style.css:131]: use tailwind class instead of css

File: style.css [hunk 28]
```diff
@@ -0,0 +1,377 @@
...
+        @apply right-0 top-0
+    }
+    .product-features{
+        background: rgba(255, 255, 255, 0.59);
+        @apply mt-20 relative overflow-hidden pt-20 flex flex-col gap-10
+    }
+    .product-features__title{
+        @apply text-5xl text-center max-w-3xl mx-auto leading-[120%]
+    }
+    .product-features::before{
```
Comment (Nkaka23dev) [inline style.css]: use tailwind class instead of custom css

**Review by Nkaka23dev — COMMENTED** [review]
Great job! only a few bugs to iron it up.

**Review by Nkaka23dev — APPROVED** [review]

**Review by alain-kubwayo — APPROVED** [review]
Good job!

File: product.html [hunk 29]
```diff
@@ -0,0 +1,221 @@
...
+        <img src="./assets/img/partner3.svg" alt="">
+        <img src="./assets/img/partner4.svg" alt="">
+        <img src="./assets/img/partner5.svg" alt="">
+    </section>
+
+    <section class="product-features">
+        <div class="product-features__main-title">Get the best out of your company with our service</div>
+        <div class="product-features__items">
+            <div class="product-features__item">
+                <img src="./assets/icons/product/feat1.svg" alt="" class="features__icon">
```
Comment (karemarsy) [inline product.html]: `features__icon` is positioned outside the Block (features).

**Review by karemarsy — COMMENTED** [review]
you should also read this notion https://musayann.notion.site/BEM-reviews-highlights-0d16ab192f134555a6f9ecf64a230498

**Comment by mercihabam** [comment]
[Merci Jacob, BEM exercise 3](https://trello.com/c/vOUWhlf5/46-merci-jacob-bem-exercise-3)

**Review by karemarsy — APPROVED** [review]

**Pull request merged**